fda-model/structures/ValueRatioWithUncertainty.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Statistics;
+using Statistics.Distributions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace structures
+{
+    public class ValueRatioWithUncertainty
+{
+        #region Fields
+        private double _standardDeviationOrMin;
+        private double _centralTendency;
+        private double _max;
+        private IDistributionEnum _distributionType;
+        #endregion
+
+        #region Constructor 
+        public ValueRatioWithUncertainty()
+        {
+            _standardDeviationOrMin = 0;
+            _centralTendency = 0;
+            _max = 0;
+            _distributionType = IDistributionEnum.Deterministic;
+        }
+        public ValueRatioWithUncertainty(IDistributionEnum distributionEnum, double standardDeviationOrMin, double centralTendency, double max = 0)
+        {
+            _distributionType = distributionEnum;
+            _standardDeviationOrMin = standardDeviationOrMin;
+            _centralTendency = centralTendency;
+            _max = max;
+        }
+        public ValueRatioWithUncertainty(double deterministicValueRatio)
+        {
+            _centralTendency = deterministicValueRatio;
+            _standardDeviationOrMin = 0;
+            _max = 0;
+            _distributionType = IDistributionEnum.Deterministic;
+        }
+        #endregion
+
+        #region Methods
+        public double Sample(double randomProbability)
```

Comment:
Should this have an argument of just probability? why call it random probability?

Reply:
you're right - it is just a probability. I've made the change.